ColorHighlighter.py

```diff
@@ -1443,7 +1443,9 @@ def restore_broken_schemes():
     g_cs = sublime.load_settings("Preferences.sublime-settings").get("color_scheme")
     for w in sublime.windows():
         for v in w.views():
-            if not os.path.exists(to_abs_cs_path(v.settings().get("color_scheme"))):
+            scheme_path = v.settings().get("color_scheme"))
```

Comment:
@Monnoroch I’m sorry and embarassed, the last bracket is unneccesary in this line and raises syntax error.
But still except the bracket, it is valid.